Source: maxice8
Upstream: no, revert of https://github.com/haasn/libplacebo/commit/bcb73f3ae243f515d5c60b5d72db5e386987ef8e
Reason: fixes compilation with our shaderc

--- src/spirv_shaderc.c
+++ src/spirv_shaderc.c
@@ -68,7 +68,7 @@ static bool shaderc_init(struct spirv_compiler *spirv)
         goto error;
 
     shaderc_compile_options_set_optimization_level(p->opts,
-            shaderc_optimization_level_performance);
+			shaderc_optimization_level_size);
 
     int ver, rev;
     shaderc_get_spv_version(&ver, &rev);
